apps/app/components/core/modals/gpt-assistant-modal.tsx

```diff
@@ -32,17 +32,17 @@ type FormData = {
   task: string;
 };
 
-const RemirrorRichTextEditor = dynamic(() => import("components/rich-text-editor"), {
-  ssr: false,
-});
-
-import { IRemirrorRichTextEditor } from "components/rich-text-editor";
-
-const WrappedRemirrorRichTextEditor = forwardRef<IRemirrorRichTextEditor, IRemirrorRichTextEditor>(
-  (props, ref) => <RemirrorRichTextEditor {...props} forwardedRef={ref} />
-);
-
-WrappedRemirrorRichTextEditor.displayName = "WrappedRemirrorRichTextEditor";
+// const RemirrorRichTextEditor = dynamic(() => import("components/rich-text-editor"), {
```

Comment:
Remove this commented code and clear it up, once tiptap editor is working fine.

Reply:
For sure!